spec/redis_counters/hash_counter_spec.rb

```diff
@@ -504,6 +504,27 @@
     end
   end
 
+  context 'when non-utf8 data contains delimeters' do
+    let(:options) do
+      {
+        counter_name:    :test_counter,
+        group_keys:      [:title, :url],
+        partition_keys:  [:date],
+        key_delimiter:   '&',
+        value_delimiter: '|'
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.